web/app/themes/dp-timber-theme/components/atoms/image/image.twig

```diff
@@ -0,0 +1 @@
+<img src="{{ src }}" alt="{{ alt }}" class="{{ classes }}" width="{{ width }}" height="{{ height }}">
```

Comment:
set a default class which will be a very generic class like classes|default('default_class_name')

Reply:
Yes added img-fluid bootstrap class, when empty class passed img-default will be there